src/components/AddressSearch.js

```diff
@@ -35,8 +35,11 @@ const defaultProps = {
     containerStyles: null,
 };
 
+// Do not convert to class component! setAddressText only works in functional components!
+// Reference: https://github.com/FaridSafi/react-native-google-places-autocomplete/issues/609#issuecomment-886133839
```

Comment:
For the record, it's not exactly accurate to say `setAddressText` only works in functional components. It can work if we translate it to a class component, but it gets much harder to make this a controlled form. Discussion about that is here: https://expensify.slack.com/archives/C03TQ48KC/p1634088400387400

I think you could change this comment to 
```
// Do not convert to class component! It's been tried before and presents more challenges than it's worth.

Reply:
Thanks for linking to that thread! I'll update the comment.